lib/new_relic/agent/instrumentation/logstasher/instrumentation.rb

```diff
@@ -0,0 +1,24 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module LogStasher
+    INSTRUMENTATION_NAME = 'LogStasher'
+
+    def self.enabled?
+      NewRelic::Agent.config[:'instrumentation.logstasher'] != 'disabled'
+    end
+
+    def build_logstash_event_with_new_relic(data, tags)
+      data = yield
```

Comment:
I think it would be best to use a different variable name here that doesn't collide with the one used for the first method argument.

Reply:
Great call out. [Updated that](https://github.com/newrelic/newrelic-ruby-agent/pull/2559/files#diff-fc9deb947cc1609417b7406320042eccc14b0a10853ab8a876ce0fc1cb4a81ceR14)